### PR TITLE
fix(#427):Admin package is not listening for the correct event

### PR DESCRIPTION
**Pull request by acenolaza**
Fixes regression bug.

Tests:
Ran e2e test suite.

Notes:
I didn't change the `fulfillWithdrawal` becuase in this particular case we are passing specific values as parameters to the filter.

### CODE REVIEW

**Comment by bbenligiray** [comment]
It feels like @Siegrift will be able to review this before I can get to it

**Comment by Siegrift** [comment]
@acenolaza why is this PR not against `master` branch?

**Comment by acenolaza** [comment]
> @acenolaza why is this PR not against `master` branch?

We are working on a big protocol refactor in beta-protocol branch and I wanted to include this fix as part of a more general refactor and fix of the admin package that's needed because of the changes done in beta-protocol.

This is the merge sequence I'm trying to follow:
427-fix-events -> fix-admin-package -> beta-protocol -> master

There is no particular reason for not doing this fix out of master and deploying it separately tho.

**Comment by Siegrift** [comment]
I would probably fix the admin package on master on my own branch and let you implement the other changes (together with simplifications) on your branch. 

I think we the after the admin package fix, we should also deploy the new version of admin package on npm.

File: packages/admin/src/implementation.ts
Comment (Siegrift) [inline packages/admin/src/implementation.ts:69]: I slightly dislike this because it sacrifices the type awareness.

Comment (Siegrift) [reply packages/admin/src/implementation.ts:69]: But I don't think there is a better way :/

**Comment by acenolaza** [comment]
> I would probably fix the admin package on master on my own branch and let you implement the other changes (together with simplifications) on your branch.
> 
> I think we the after the admin package fix, we should also deploy the new version of admin package on npm.

Cool, then I'll wait for your fix and merge this one (if necessary) with fixes due to contract refactor afterwards.

**Comment by acenolaza** [comment]
Applied fix [#455](https://github.com/api3dao/airnode/commit/8a193fb97c17e79d2b13d51baa3d8084def8e054) to this branch and it should now be ready for merge

**Review by Siegrift — APPROVED** [review]
LGTM

**Comment by Siegrift** [comment]
I guess the failed tests are because other packages are not up to date with the protocol?

**Comment by acenolaza** [comment]
> I guess the failed tests are because other packages are not up to date with the protocol?

Yes, that's correct. I'm trying to fix one package at a time. I hope to get admin package fix right after merging this branch and then continue with operation package probably

**Pull request merged**
